.github/workflows/end-to-end-test.yml

```diff
@@ -0,0 +1,45 @@
+name: End-To-End Test
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
![image](https://user-images.githubusercontent.com/37993418/177978841-b2d4d2ad-e84f-4313-b7ca-ed523b7f34e5.png)

Reply:
This took a few hours of my life.